.github/workflows/e2e.yml

```diff
@@ -0,0 +1,122 @@
+name: e2e
+
+on: [push]
+env:
+  REGISTRY: 'ghcr.io/dell/csi-baremetal'
```

Comment:
In this step, we didn't add anything to the registry, so it doesn't matter what name it has.

Reply:
Why do we need to have `REGISTRY` variable at all if not used?

Reply:
You have no default values for this parameter. This means that I cannot leave it blank - otherwise there will be a problem with `docker tag`

Reply:
ok